### PR TITLE
Add entry in faq for manifests lists

### PR DESCRIPTION
**Pull request by loosebazooka**
Add faq entry, so we can link to it for failing builds.

Using example provided in: #2148

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -466,6 +467,38 @@ Some plugins, such as the [Docker Prepare Gradle Plugin](https://github.com/gcla
 
 To inspect the image that is produced from the build using Docker, you can use commands such as `docker inspect your/image:tag` to view the image configuration, or you can also download the image using `docker save` to manually inspect the container image. Other tools, such as [dive](https://github.com/wagoodman/dive), provide nicer UI to inspect the image.
 
+### How do I specify an image from a manifest list?
+
+By design, Jib will pick always select an image for the platform:`amd64/linux` if the target image reference is a [manifest list](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list). If you need to specify a different image from a manifest list you must specify the image digest for the platform you are targeting. 
+
+To determine the digest of an image in a manifest list, you can view the manifest list with:
+```
+$ docker manifest inspect openjdk:8
```
Comment (chanseokoh) [inline docs/faq.md:476]: The `manifest inspect` command is an experimental one and needs to update `~/.docker/config.json`. Maybe we can add some links:
https://docs.docker.com/engine/reference/commandline/manifest_inspect/ https://docs.docker.com/engine/reference/commandline/cli/#experimental-features

File: docs/faq.md [hunk 2]
```diff
@@ -466,6 +467,38 @@ Some plugins, such as the [Docker Prepare Gradle Plugin](https://github.com/gcla
 
 To inspect the image that is produced from the build using Docker, you can use commands such as `docker inspect your/image:tag` to view the image configuration, or you can also download the image using `docker save` to manually inspect the container image. Other tools, such as [dive](https://github.com/wagoodman/dive), provide nicer UI to inspect the image.
 
+### How do I specify an image from a manifest list?
+
+By design, Jib will pick always select an image for the platform:`amd64/linux` if the target image reference is a [manifest list](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list). If you need to specify a different image from a manifest list you must specify the image digest for the platform you are targeting.
```
Comment (chanseokoh) [inline docs/faq.md:472]: "will pick always select" --> "will always select"
"for the platform:`amd64/linux`" --> "for the platform `amd64/linux`

File: docs/faq.md [hunk 3]
```diff
@@ -27,7 +27,8 @@ If a question you have is not answered below, please [submit an issue](/../../is
 [What would a Dockerfile for a Jib-built image look like?](#what-would-a-dockerfile-for-a-jib-built-image-look-like)\
 [How can I inspect the image Jib built?](#how-can-i-inspect-the-image-jib-built)\
 [I would like to run my application with a javaagent.](#i-would-like-to-run-my-application-with-a-javaagent)\
-[How can I tag my image with a timestamp?](#how-can-i-tag-my-image-with-a-timestamp)
+[How can I tag my image with a timestamp?](#how-can-i-tag-my-image-with-a-timestamp)\
+[How do I specify an image from a manifest list?](#how-do-i-specify-an-image-from-a-manifest-list)
```
Comment (chanseokoh) [inline docs/faq.md:31]: For the title, how about "a manifest list or an OCI index"? (But we can omit OCI index in the text for brevity.)

And how about "specify OS and architecture of a base image from..."?

File: docs/faq.md [hunk 4]
```diff
@@ -466,6 +467,38 @@ Some plugins, such as the [Docker Prepare Gradle Plugin](https://github.com/gcla
...
+### How do I specify an image from a manifest list?
+
+By design, Jib will pick always select an image for the platform:`amd64/linux` if the target image reference is a [manifest list](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list). If you need to specify a different image from a manifest list you must specify the image digest for the platform you are targeting. 
+
+To determine the digest of an image in a manifest list, you can view the manifest list with:
+```
+$ docker manifest inspect openjdk:8
+```
+
+You can then inspect the output for the specific image you want (in this example an image for the platform:`arm64/linux`)
```
Comment (chanseokoh) [inline docs/faq.md:479]: "for the platform:`amd64/linux`" --> "for the platform `amd64/linux`

File: docs/faq.md [hunk 5]
```diff
@@ -466,6 +467,38 @@ Some plugins, such as the [Docker Prepare Gradle Plugin](https://github.com/gcla
...
+            "architecture": "arm64",
+            "os": "linux",
+            "variant": "v8"
+         }
+      }
+   ]
+}
+```
+
+You then use the digest for the target platform you desire in your `jib.to.image` in the form `"opendjdk:8@sha256:1fbd49e3fc5e53154fa93cad15f211112d899a6b0c5dc1e8661d6eb6c18b30a6"`
```
Comment (chanseokoh) [inline docs/faq.md:501]: Can we add something like

"As in the example, you can preserve the tag for commenting purposes, but note that the tag will not be used but ignored."

Comment (loosebazooka) [reply docs/faq.md]: updated to show both. both are valid?

File: docs/faq.md (hunk 3 above)
Comment (loosebazooka) [reply docs/faq.md:31]: I'm not really sure about the title.

`How do I specify a platform in the manifest list (or OCI index) of a base image?"

Comment (chanseokoh) [reply docs/faq.md:31]: Yeah, I think "platform" is a good keyword for search.

File: docs/faq.md [hunk 6]
```diff
@@ -466,6 +467,38 @@ Some plugins, such as the [Docker Prepare Gradle Plugin](https://github.com/gcla
 
 To inspect the image that is produced from the build using Docker, you can use commands such as `docker inspect your/image:tag` to view the image configuration, or you can also download the image using `docker save` to manually inspect the container image. Other tools, such as [dive](https://github.com/wagoodman/dive), provide nicer UI to inspect the image.
 
+### How do I specify an image from a manifest list?
+
+By design, if the target image reference is a [manifest list](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list), Jib will always select an image for the platform `amd64/linux` . If you need to specify a different image from a manifest list you must specify the digest for the platform you are targeting. 
+
+To view a manifest, [enable experimental docker cli](https://docs.docker.com/engine/reference/commandline/cli/#experimental-features) features and then run the [manifest inspect](https://docs.docker.com/engine/reference/commandline/manifest_inspect/) command.
```
Comment (chanseokoh) [inline docs/faq.md:474]:
```suggestion
To view a manifest, [enable experimental docker CLI](https://docs.docker.com/engine/reference/commandline/cli/#experimental-features) features and then run the [manifest inspect](https://docs.docker.com/engine/reference/commandline/manifest_inspect/) command.
```

File: docs/faq.md [hunk 7]
```diff
@@ -466,6 +467,38 @@ Some plugins, such as the [Docker Prepare Gradle Plugin](https://github.com/gcla
 
 To inspect the image that is produced from the build using Docker, you can use commands such as `docker inspect your/image:tag` to view the image configuration, or you can also download the image using `docker save` to manually inspect the container image. Other tools, such as [dive](https://github.com/wagoodman/dive), provide nicer UI to inspect the image.
 
+### How do I specify an image from a manifest list?
+
+By design, if the target image reference is a [manifest list](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list), Jib will always select an image for the platform `amd64/linux` . If you need to specify a different image from a manifest list you must specify the digest for the platform you are targeting.
```
Comment (chanseokoh) [inline docs/faq.md:472]:
```suggestion
By design, if the target image reference is a [manifest list](https://docs.docker.com/registry/spec/manifest-v2-2/#manifest-list), Jib will always select an image for the platform `amd64/linux`. If you need to specify a different image from a manifest list you must specify the digest for the platform you are targeting.
```

Comment (chanseokoh) [reply docs/faq.md:472]: We shouldn't forget to update this paragraph once we fix #1567. But I think we will forget, so should we add a link to the issue?

Comment (loosebazooka) [reply docs/faq.md]: I added a todo on the bug.

**Pull request merged**